electrum/invoices.py

```diff
@@ -137,6 +137,10 @@ def get_expiration_date(self):
         # 0 means never
         return self.exp + self.time if self.exp else 0
 
+    def has_expired(self):
+        exp = self.get_expiration_date()
+        return exp and exp < time.time()
```

Comment:
Let's make sure this method returns a `bool`.
```
>>> 0 and (0 < 5)
0
```

```suggestion
    def has_expired(self) -> bool:
        exp = self.get_expiration_date()
        return bool(exp) and exp < time.time()
```

Reply:
fixed